### DISCUSSION TITLE
Messages during the startup

### DISCUSSION THREAD

**Discussion by OrkunTokdemir** [microsoft/vscode-discussions#2238] [Extension Development QnA] [answered]
I noticed that vscode restricts the number of messages we can show during the startup. It can only show up to 3 messages for all extensions. It isn't a reliable way to communicate with users. Is there any way to increase it or a workaround?

Only the first 3 messages will be shown when `vscode` starts.

Example code: 
```typescript
export async function activate(context: vscode.ExtensionContext) {
  void vscode.window.showInformationMessage('Message 1111111');
  void vscode.window.showInformationMessage('Message 2222222');
  void vscode.window.showInformationMessage('Message 3333333');
  void vscode.window.showInformationMessage('Message 4444444');
  void vscode.window.showInformationMessage('Message 5555555');
  void vscode.window.showInformationMessage('Message 6666666');
  void vscode.window.showInformationMessage('Message 7777777');
}
```

**Comment by olguzzar** [comment, marked as answer]
Hi @OrkunTokdemir, while only 3 messages will pop up at a time, if the user clicks their notifications, they should see all of the messages listed there

**Reply by OrkunTokdemir** [reply]
It is a bit hard to notice the notification button, especially with the extra little dot indicating unread notifications. Still, it is a valid answer. Thanks.
